rosidl_generator_c/resource/action__type_support.h.em

```diff
@@ -11,7 +11,7 @@
 // Forward declare the get type support functions for this type.
 ROSIDL_GENERATOR_C_PUBLIC_@(package_name)
 const rosidl_action_type_support_t *
-  ROSIDL_TYPESUPPORT_INTERFACE__ACTION_SYMBOL_NAME(
+ROSIDL_TYPESUPPORT_INTERFACE__ACTION_SYMBOL_NAME(
```

Comment:
This change shouldn't be part of the PR since the change is already present on the target branch. This probably needs to be rebased correctly.

Same below.

Reply:
The same comment still applies to the message / service macros.